src/javascripts/ng-admin/Crud/list/ListLayoutController.js

```diff
@@ -0,0 +1,50 @@
+/*global define*/
+
+define(function () {
```

Comment:
Can you use more ES6 compliant syntax in all new files? Just remove the define and `export defaults` instead of `return`